.github/workflows/e2e-tests-ondemand.yaml

```diff
@@ -4,12 +4,9 @@ on:
   schedule:
     # every Mon/Wed/Fri at 08:00 UTC
     - cron: 0 8 * * 1,3,5
-  workflow_dispatch:
-    inputs:
-      target:
-        description: Target branch to run E2E tests over
-        required: true
-        default: release-1.3
+
+env:
+  branch: 'release-1.3'
```

Comment:
the only problem with this approach, if I want to trigger hotfix, which is not yet merged to release branch, i can't.

Reply:
you should not run hot fix on all pipelines imho

Reply:
okey anyway let's try with this fix.